### ISSUE TITLE
Twitter Account

### ISSUE THREAD

**Issue by SomMeri** [less/less-meta#3]
It might be useful to have a twitter account and maybe use some common hashtag (#lesscss?). It could be used for 
- new release announcements, 
- new feature announcements, 
- occasional tip&trics / articles / doc updates,
- active discussions when we want more feedback,
- etc.

As far as I know, there is no feed you could listen to and instantly know about these things. I usually learn about new release when someone mentions it on unrelated issue. 

It does not have to tweet often, only when something important happen. It may also help with community building, twitter is less formal then github issues. I have such account for less4j and it is minimum overhead.

**Comment by matthew-dean** [comment]
https://twitter.com/lesstocss  - Who should I give access to on Tweetdeck?

**Comment by SomMeri** [comment]
Can it be multiuser?

**Comment by matthew-dean** [comment]
Yes, Tweetdeck (owned by Twitter), allows you to grant posting permission to multiple other Twitter accounts.

**Comment by matthew-dean** [comment]
@SomMeri Did you want to do some tweeting about / for Less.js? I could add you in Tweetdeck to this Twitter account.

**Comment by SomMeri** [comment]
@matthew-dean Sorry for long delay in activity, I got a new job and was unable to follow or do pretty much anything.  I will see whether I am able to follow development and if yes, I could tweet about upcoming features (like I do for less4j).

**Comment by matthew-dean** [comment]
@SomMeri Cool! I can add you in Tweetdeck. What's your Twitter username?

**Comment by SomMeri** [comment]
@matthew-dean It is [SomMeri](https://twitter.com/SomMeri). Twitter required at least 6 characters, just as github :)
